TicketBookingAPI/Controller/AircraftController/AircraftController.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using TicketBooking.Data;
+using TicketBooking.Service.AircraftService;
+using TicketBooking.Data.DbContext;
+
+namespace TicketBookingAPI.Controller.AircraftController
+{
+    [Route("api/[Controller]")]
+    [ApiController]
+    public class AircraftController : ControllerBase
+    {
+        private readonly AircraftService _service;
+
+        public AircraftController(AircraftService service)
+        {
+            _service = service;
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> AddAircraft(AircraftDTO aircraftDTO)
```

Comment:
Should use another model name, it can be `AircraftViewModel` or `AircraftRequestModel`.................